lib-rs/src/lib.rs

```diff
@@ -19,6 +80,27 @@ use std::fmt;
 /// Same as [`slice::from_raw_parts`].
 ///
 /// [`slice::from_raw_parts`]: https://doc.rust-lang.org/std/slice/fn.from_raw_parts.html
+///
+/// Example
```

Comment:
I think it's nice to use a header for the Examples in each section of the docs. This is pretty common in other projects I've seen. Also, use Examples instead of Example if it's a heading (just to standardize on a single heading which covers one or more examples).

```suggestion
/// # Examples
```